src/configure

```diff
@@ -6871,7 +6871,7 @@ if test "$VECTOR" = "_papi_dummy_vector"; then
 		.num_native_events = 0,
 		.num_preset_events = 0,
 		.num_cntrs = 0,
-		.name = \"Your system is unsupported! \",
+		.name = \"No Components Configured. \",
```

Comment:
Be sure to make this change in configure.in, as well.

Reply:
I have made the change to configure.in. I keep forgetting to modify the configure.in file as well, I am very sorry.